src/components/Locations.js

```diff
@@ -0,0 +1,53 @@
+import "../App.css";
+import React from "react";
+
+
+class Locations extends React.Component {
+  constructor(){
+    super()
+
+    this.state = {
+      locations:[]
+      // currentLocation: null,
+    };
+  }
+    fetchLocations = () =>{
+        fetch("https://ghibliapi.herokuapp.com/locations")
+        .then((res)=> res.json())
+        .then((data) => {
+          this.setState({
+            locations: data,
+            
+          });
+       });
+   };
+
+   componentDidMount = () =>{
+     this.fetchLocations();
+   }
+
+
+    // this.setState({
+    //   currentLocation: currentLocationObj,
+    // });
+  // };
+
+
+    render(){
+
+
+
+  return (
+    <div className="locations">
+      <ul className= "list-of-locations">
+        {this.state.locations.name}
```

Comment:
```suggestion
        {this.state.locations.map((location) => {
        return (
        <li className="location" key={location.id}>
          <h2>Name: {location.name}</h2>  //<-- each individual object has a .name 
          <h3>Climate: {location.climate}</h3>  
          <h3>Terrain: {location.terrain}</h3> 
        </li>
        )
        })}
```
`this.state.locations` is an entire array of objects, so you can't access the `name` property of each location using dot notation like this. You need to map through the entire array of objects and return JSX displaying the name, climate, and terrain for each one.